todomvrx/src/main/java/com/airbnb/mvrx/todomvrx/views/TaskItemView.kt

```diff
@@ -14,7 +14,9 @@ import com.airbnb.mvrx.todomvrx.todoapp.R
 
 @ModelView(autoLayout = ModelView.Size.MATCH_WIDTH_WRAP_HEIGHT)
 class TaskItemView @JvmOverloads constructor(
-        context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
+    context: Context,
```

Comment:
To be honest, I hate that `ktlint` forces to have these in separate lines. The auto formatter and formatting validator in IntelliJ (with Kotlin plugin) does not do that and allow all params to be on the same line as long as they are short.

Reply:
I cannot do it. Feel free to resolve this one. :)